ports/ghostris/Ghostris.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+
+export PORT_32BIT="Y"
```

Comment:
Is this a 32 bit port? If not needs to be removed

Reply:
It is 32bit, but I can redo it in on gmloader if you want it easily distinguishable from the 64bit ones.